chart/templates/98-nginx-proxy.yml

```diff
@@ -81,10 +81,19 @@ spec:
               name: process-env
           - configMapRef:
               name: ssl-env
+          - configMapRef:
+              name: opensearch-env
```

Comment:
This is correct, nginx now needs these variables as well to be able to determine if we're in "opensearch mode" or "elasticsearch mode"